src/Components/Browser.JS/src/package-lock.json

```diff
@@ -16,7 +16,7 @@
       "integrity": "sha512-Q/rSs/UDzw0iwhqZILzMMP0m/lTVNSQDS9aNkBTkt8aAAKAEMriCMTy0Ahx9xPuMTzFtyPRjNgEZfEkWUHajcg==",
       "dev": true,
       "requires": {
-        "msgpack5": "4.2.0"
+        "msgpack5": "^4.0.2"
```

Comment:
I just committed what `npm install` said was the right restore result. Just a guess - this project has only every been using `npm ci` in local builds, so the lock file has drifted from what the `package.json` fail says to install. cc @SteveSandersonMS

Reply:
After looking at this closer, I'm not sure if NPM is doing the right thing or not. I'll leave this for the MVC team to look into later since it's unrelated to what I'm trying to accomplish here.